sunbeam-python/sunbeam/plugins/validation/plugin.py

```diff
@@ -412,36 +452,10 @@ def configure_validation(self, options: Optional[List[str]] = None) -> None:
         )
 
     @click.command()
-    @click.option(
-        "-r",
-        "--regex",
-        default="",
-        help=(
-            "A list of regexes, whitespace separated, used to select tests from"
-            " the list."
-        ),
-    )
-    @click.option(
-        "-e",
-        "--exclude-regex",
-        default="",
-        help="A single regex to exclude tests.",
-    )
-    @click.option(
-        "-t",
-        "--serial",
-        is_flag=True,
-        default=False,
-        help="Run tests serially. By default, tests run in parallel.",
-    )
-    @click.option(
-        "-l",
-        "--test-list",
-        default="",
-        help=(
-            "Use a predefined test list. See `sunbeam validation test-lists`"
-            " for available test lists."
-        ),
+    @click.argument(
+        "profile",
+        default=DEFAULT_PROFILE.name,
+        type=click.Choice(PROFILES.keys()),
```

Comment:
If we have the detailed information about all available profiles showing under `sunbeam validation profiles`, we don't need to list the options here. Instead, we can add `metavar='<profile>'` so the usage in the help message shows:
```
Usage: sunbeam validation run [OPTIONS] <profile>
```

This complements the suggested change in my other comment.

Reply:
not a blocker: maybe `metavar=[PROFILE=refstack]` can make the default value stands out more